src/dungeon_2.c

```diff
@@ -8,9 +8,24 @@
 extern const char gUnknown_8108F10[];
 extern const char gUnknown_8108F18[];
 extern const char gUnknown_8108F2C[];
+extern struct DungeonLocation gUnknown_8107828[];
 extern u8 gDungeonFloorCount[];
 extern u8 gUnknown_81077E8[];
 
+// this one just does not want to get right
+// int sub_809017C(int a1) {
```

Comment:
Non-matching code like this can be surrounded with a `NONMATCHING` block, using a `NAKED` function otherwise. Example: https://github.com/pret/pmd-red/blob/master/src/dungeon_ai_movement.c#L702

Reply:
What problem is causing this not to be matched? Was able to match it just find with CE and then pasted just fine into the file (and matched locally).

https://staging.cexplore.karathan.at/z/OBlVCK

Reply:
This was one of the earlier ones I did getting back into Pret, I was using wrong settings for agbcc back then I believe, so probably that was the cause, did you update the file?

Reply:
No, I haven't but feel free to just copy the code and commit it.